### DISCUSSION TITLE
Save the position and zoom where you left off in the excalidraw

### DISCUSSION THREAD

**Discussion by animalice** [zsviczian/obsidian-excalidraw-plugin#1833] [Ideas]
I think it would be such a good feature to be able to leave the exact and precise position/zooming where I left off in the Excalidraw instead of simply seeing everything I did in the distance whenever I refresh it. It would save me so much time, instead I have to keep zooming in where I left off every time I open an Excalidraw, it's so time-consuming. It could have some option to turning this save "position feature" on and off...  Sorry if I'm in the wrong place for posting this.

**Comment by Barmaid1076** [comment]
that would be a great feature
